README.md

```diff
@@ -10,39 +10,44 @@ The program intercepts and pretends to be a kahoot client. After receiving quiz
 
 ## Installation
 - [ ] Optional: Make a Kahoot Account if you don't have the Kahoot's ID and want to search for a Kahoot by name
-- [ ] Install Python3
+- [ ] Install Python3.9
 	
   On Windows visit `https://www.python.org/downloads/windows/`
-	- Click latest Python 3 release
+	- Click latest Python 3.9 release
 	- Scroll down to the bottom to the section titled "Files"
 	- Click the Windows Installer (64-bit) link to download the ".exe"
 	- In File Explorer right click the file and click "Run as Administrator"
-	- Check the boxes "Install launcher for all users (recommended)" and "Install Python 3.X to path"
+	- Check the boxes "Install launcher for all users (recommended)" and "Install Python 3.9 to path"
 	
 	On macOS 11+ (Intel) and macOS 11+ (Apple Sillicon) visit "`https://www.python.org/downloads/macos/`"
-	- Click latest Python 3 release
+	- Click latest Python 3.9 release
 	- Scroll down to the bottom to the section titled "Files"
 	- Click the macOS 64-bit universal2 installer link to download the ".pkg"
 	- Run the downloaded ".pkg"
 	
     On Debian GNU/Linux 11+ based distros:
-    - `sudo apt install python3 python3-pip`
+    - `sudo apt update
+    - `sudo apt install build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libsqlite3-dev libreadline-dev libffi-dev curl libbz2-dev`
+    - `wget https://www.python.org/ftp/python/3.9.7/Python-3.9.7.tgz`
+    - `tar -xf Python-3.9.7.tgz`
+    - `cd Python-3.9.7`
+    - `./configure --enable-optimizations`
+    - `make -j 2`
```

Comment:
Use all but 2 cores instead of 2 cores, that'll speed up things by a lot for people.